docs/src/user-guide/format-struct-logs-syntax.md

```diff
@@ -95,4 +102,35 @@ The formatted string will be:
 2015-03-23 19:29:48.942 INFO {0} latency=56 org.apache.hadoop.metrics2.impl.MetricsConfig: loaded properties from hadoop-metrics2.properties
 ```
 
-For a list of currently supported formatters, see [Formatters](format-struct-logs-formatters).
+### Key-Value Pair IR Example
+
+Consider the following Key-Value Pair IR log event:
+```
+{
+  "auto-generated": {
+    "ts": 1741371422000
+  },
+  "user-generated": {
+    "message": "Callback registered to fire in 5 seconds:",
+    "ts": 1741371427000
+  }
+}
+```
+
+We can format this using the following YScope format string:
+
+```
+{@ts:timestamp} {message} {ts:timestamp}
+```
+
+* In the first placeholder, we have the auto-generated field name `@ts`. The `@` prefix
+  specifies the field is from the auto-generated namespace.
+* The second and third placeholders refer to the message and timestamp fields in the
+  user-generated namespace, respectively.
+
+The formatted string will be:
+```
+2025-03-07T18:17:02Z Callback registered to fire in 5 seconds: 2025-03-07T18:17:07Z
+```
+
+For a list of currently supported formatters, see [Formatters](format-struct-logs-formatters).
```

Comment:
Since this applies to both examples, let's move this to the end of the `formatter-name` section instead.